sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -624,7 +624,7 @@ export class StudentController extends BaseController {
     // files (saved during the upload) are update to its proper group,file_origin
     //  and add the metadata (if available)
     await this.fileService.updateStudentFiles(
-      existingStudent.id,
+      existingStudent,
```

Comment:
why did we make this change?

Reply:
Previously when we update the students file on submit documents, we just inserted records, now we are sending email notification to the SBC, to process on the files. The email subject needs additional data like the student -> lastname, given name, dob etc, so reusing the student object from the previous controller call.